src/App.js

```diff
@@ -0,0 +1,76 @@
+import './App.css';
+
+function App() {
+  const name = "Cindy";
+  var nameComponent = (name === "Cindy")?
+  <h1 className="darkpurple-text"> My name is {name}!</h1> : <p> My name is not Cindy.</p>
+  const buttonFunction = () =>
+    console.log("The button was clicked.");
+    <button onClick="window.open='https://www.youtube.com/watch?v=rEq1Z0bjdwc';"></button>
```

Comment:
nice! this was a good idea. not sure that it will work, so in this case it would be best to use an `<a>` tag. after tonights class we'll see how we can use buttons in a more intuitive way. but good job including it here